### ISSUE TITLE
Spawn Zone = Safe Zone

### ISSUE THREAD

**Issue by eLudet** [peace-maker/smrpg#331]
Hey there.

Is there by any means a possibility that there could be a convar making spawn areas safe zones? We have multiple people asking for this.

Maybe there is a plugin existing, that could correspond with your plugin, but so far I have not found any.

**Comment by WanekWest** [comment]
For example: https://hlmod.ru/resources/easy-spawn-protector.643/

**Comment by peace-maker** [comment]
What are safe zones in your opinion? There's no easy way to know where the spawn areas are.

When I used to host servers I've used the plugin I've just uploaded to define zones on the maps where players wouldn't earn any experience if they stand in them or their target stands in one. Maybe this is what you're after. Spawn protection aka no damage after spawn is a different story and covered by other plugins I guess.

The added `smrpg_noxpzones` plugin uses my [mapzonelib](https://github.com/peace-maker/mapzonelib) plugin which handles the creation of zones on the map, so you'll have to install that too if you want to use it.
